pkl-cli/src/test/kotlin/org/pkl/cli/CliEvaluatorTest.kt

```diff
@@ -1206,6 +1214,196 @@ result = someLib.x
       .hasMessageNotContainingAny("java.", "sun.") // class names have been filtered out
   }
 
+  @Test
+  fun `eval http module from proxy`(wwRuntimeInfo: WireMockRuntimeInfo) {
+    stubFor(
+      get(urlEqualTo("/bar.pkl")).withHost(equalTo("not.a.valid.host")).willReturn(ok("foo = 1"))
+    )
+    val options =
+      CliEvaluatorOptions(
+        CliBaseOptions(
+          sourceModules = listOf(URI("http://not.a.valid.host/bar.pkl")),
+          proxyAddress = URI("http://localhost:${wwRuntimeInfo.httpPort}"),
+          allowedModules = SecurityManagers.defaultAllowedModules + Pattern.compile("http:"),
+        ),
+      )
+    val output = evalToConsole(options)
+    assertThat(output).isEqualTo("foo = 1\n")
+  }
+
+  @Test
+  fun `eval https -- no proxy`(wwRuntimeInfo: WireMockRuntimeInfo) {
+    // pick an address on the local machine so we can be sure this test is not making any outbound
+    // connections.
+    val openPort = ServerSocket(0).use { it.localPort }
+    val targetAddress = "https://127.0.0.1:$openPort"
+    val options =
+      CliEvaluatorOptions(
+        CliBaseOptions(
+          // use loopback address to prevent test from making outbound http connection.
+          sourceModules = listOf(URI("$targetAddress/foo.pkl")),
+          proxyAddress = URI(wwRuntimeInfo.httpBaseUrl),
+          noProxy = listOf("*"),
+          allowedModules = SecurityManagers.defaultAllowedModules + Pattern.compile("http:"),
+        )
+      )
+    assertThatCode { evalToConsole(options) }
+      .hasMessageContaining("I/O error loading module `$targetAddress/foo.pkl`")
+  }
+
+  @Test
+  @Disabled // TODO: Re-enable after certification logic is fixed.
```

Comment:
[nit] this is a little clearer.

```suggestion
  @Disabled // TODO: figure out why this is failing.
```